mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -179,19 +212,31 @@ CLASS aif net/minecraft/entity/Entity
 		ARG 2 state
 	METHOD a readCustomDataFromTag (Lib;)V
 		ARG 1 tag
+	METHOD a hasPassengerType (Ljava/lang/Class;)Z
+		ARG 1 clazz
 	METHOD a addScoreboardTag (Ljava/lang/String;)Z
+		ARG 1 tag
 	METHOD a setUuid (Ljava/util/UUID;)V
+		ARG 1 uuid
 	METHOD a onTrackedDataSet (Lqh;)V
 		ARG 1 data
+	METHOD a canBeSpectated (Lvh;)Z
+		ARG 1 spectator
 	METHOD a playSound (Lyj;FF)V
 		ARG 1 sound
 		ARG 2 volume
+		ARG 3 pitch
 	METHOD a isInFluid (Lzb;)Z
+		ARG 1 fluidTag
 	METHOD a isInFluid (Lzb;Z)Z
+		ARG 1 fluidTag
 		ARG 2 requireLoadedChunk
 	METHOD a toListTag ([D)Lii;
+		ARG 1 values
 	METHOD a toListTag ([F)Lii;
+		ARG 1 values
 	METHOD aA spawnSprintingParticles ()V
+	METHOD aB touchLava ()V
```

Comment:
touchLava or onLavaTouch or?

Reply:
It doesn't handle anything that happens when touching lava, `tick()` is doing that.
```java
public void touchLava() {
    this.touchingLava = true;
}
```

Reply:
`setTouchingLava()`. But maybe it should be `inLava`, the same way we have `inWater`?

Reply:
`set` implies it has a parameter.

Reply:
No, there's lots of set methods with no parameters. For booleans, `set` and `unset` set to true and false.

Reply:
All boolean set methods I know (in general) use `set` with a parameter, and a command-tone for changing them. to true or false. I think that shows better what they do. I can change it, if it's wanted, though.

Reply:
There are many `set` methods for booleans with no parameter in yarn, and `set` and `unset` are common names for setting to true and false. If you don't like set, you can use `mark`. `touchLava` makes it seem like the player was not touching lava and will be moved into the lava to touch it.